$NetBSD: patch-az,v 1.2 2005/12/01 18:13:24 wiz Exp $

@dircategory and @direntry are what makeinfo turns into the
INFO-DIR-SECTION block that install-info reads.  Without them a rebuilt
info file drops out of the dir node again, undoing by hand what the
patches against info/ put there.

--- man/sc.texi.orig	1994-01-08 09:41:40.000000000 +0100
+++ man/sc.texi
@@ -6,6 +6,13 @@
 @iftex
 @finalout
 @end iftex
+
+@dircategory Editors
+@direntry
+* SC: (sc).		Supercite lets you cite parts of messages you're
+			  replying to, in flexible ways.
+@end direntry
+
 @c @setchapternewpage odd		% For book style double sided manual.
 @comment %**end of header (This is for running Texinfo on a region.)
 @c      @smallbook
